src/main/java/com/mcmoddev/orespawn/data/Constants.java

```diff
@@ -0,0 +1,10 @@
+package com.mcmoddev.orespawn.data;
+
+public class Constants {
+	public static final String MODID = "orespawn";
+	public static final String NAME = "MMD OreSpawn";
+	public static final String VERSION = "3.0.0.0";

```

Comment:
Should be 3.0.0, not 3.0.0.0 (The last number is the jenkins build number and not counted as part of the actual mod version)